src/components/PeopleFilters.tsx

```diff
@@ -35,40 +109,50 @@ export const PeopleFilters = () => {
           <div className="level-left">
             <a
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              className={classNames('button mr-1', {
+                'is-info': centuries.includes('16'),
+              })}
+              onClick={() => handleCenturiesChange('16')}
             >
               16
             </a>
 
             <a
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
+              className={classNames('button mr-1', {
+                'is-info': centuries.includes('17'),
+              })}
+              onClick={() => handleCenturiesChange('17')}
             >
               17
             </a>
```

Comment:
hardcode, if you need to add 12,13 do you still copy and paste this part of code? 
Try to use iteration